boilerplate/openshift/golang-osd-operator-osde2e/standard.mk

```diff
@@ -69,7 +69,6 @@ e2e-harness-generate:
 e2e-harness-build: GOFLAGS_MOD=-mod=mod
 e2e-harness-build: GOENV=GOOS=${GOOS} GOARCH=${GOARCH} CGO_ENABLED=0 GOFLAGS="${GOFLAGS_MOD}"
 e2e-harness-build:
-	go mod tidy
```

Comment:
I'm not sure if this will work without at least a `go mod download`, which may run into the same behavior